packages/web-awesome/src/stores/layout.ts

```diff
@@ -0,0 +1,39 @@
+import { getReportOptions } from "@allurereport/web-commons";
+import { computed, signal } from "@preact/signals";
+import type { AllureAwesomeReportOptions } from "../../types.js";
+
+type Layout = "base" | "split";
```

Comment:
Make the type available in the entire application because the type can be used in the Plugin Options type for better consistency